integration_tests/integration/probationPractitionerReferrals.spec.js

```diff
@@ -77,45 +77,66 @@ describe('Probation practitioner referrals dashboard', () => {
   it('user views an end of service report', () => {
     cy.stubGetSentReferralsForUserToken([])
     cy.login()
-
-    const serviceCategory = serviceCategoryFactory.build()
-    const intervention = interventionFactory.build({ serviceCategories: [serviceCategory] })
+    const serviceCategory1 = serviceCategoryFactory.build()
+    const serviceCategory2 = serviceCategoryFactory.build({
+      desiredOutcomes: [
+        {
+          id: '2d63bedc-9658-40c4-9d31-da649396ace1',
+          description: 'Some other desired outcome',
+        },
+      ],
+    })
+    const intervention = interventionFactory.build({ serviceCategories: [serviceCategory1, serviceCategory2] })
     const deliusServiceUser = deliusServiceUserFactory.build()
     const referral = sentReferralFactory.build({
       referral: {
         interventionId: intervention.id,
-        serviceCategoryIds: [serviceCategory.id],
-        desiredOutcomesIds: [serviceCategory.desiredOutcomes[0].id],
+        serviceCategoryIds: [serviceCategory1.id, serviceCategory2.id],
+        desiredOutcomesIds: [serviceCategory1.desiredOutcomes[0].id],
```

Comment:
Can we remove `desiredOutcomesIds` here now?

Reply:
thanks I've removed it here - as it's connected to PP